src/include/module/module/interface.h

```diff
@@ -73,12 +73,13 @@ struct sof_sink;
 struct module_interface {
 	/**
 	 * Module specific initialization procedure, called as part of
-	 * module_adapter component creation in .new()
+	 * module_adapter component creation in .new(). Usually can be __cold
```

Comment:
Can you add some general context above the ```struct module_interface``` so that it makes sense for each op.
i.e.
"Module operations can be optimized for performance (default - no action) or for memory and power efficiency (opt in using __cold). Recommended that module authors review their modules for non time sensitive code and mark using __cold based on the descriptions below. This will ensure modules maintain peak performance and peak power/memory efficient. Also see fast_get() API for same efficiencies for DATA."